libs/libmbim/Makefile

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (C) 2016 Velocloud Inc.
+# Copyright (C) 2016 Aleksander Morgado <aleksander@aleksander.es>
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libmbim
+PKG_VERSION:=1.18.2
```

Comment:
1.20.0 is out
https://www.freedesktop.org/software/libmbim/libmbim-1.20.0.tar.xz